pkg/yurtappmanager/webhook/yurtappdaemon/mutating/yurtappdaemon_create_update_handler.go

```diff
@@ -0,0 +1,76 @@
+package mutating
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"k8s.io/klog"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+
+	unitv1alpha1 "github.com/openyurtio/yurt-app-manager/pkg/yurtappmanager/apis/apps/v1alpha1"
+	"github.com/openyurtio/yurt-app-manager/pkg/yurtappmanager/util"
+	webhookutil "github.com/openyurtio/yurt-app-manager/pkg/yurtappmanager/webhook/util"
+)
+
+// YurtAppDaemonCreateUpdateHandler handles UnitedDeployment
+type YurtAppDaemonCreateUpdateHandler struct {
+	// To use the client, you need to do the following:
+	// - uncomment it
+	// - import sigs.k8s.io/controller-runtime/pkg/client
+	// - uncomment the InjectClient method at the bottom of this file.
+	Client client.Client
+
+	// Decoder decodes objects
+	Decoder *admission.Decoder
+}
+
+var _ webhookutil.Handler = &YurtAppDaemonCreateUpdateHandler{}
+
+func (h *YurtAppDaemonCreateUpdateHandler) SetOptions(options webhookutil.Options) {
+	h.Client = options.Client
+	return
+}
+
+// Handle handles admission requests.
+func (h *YurtAppDaemonCreateUpdateHandler) Handle(ctx context.Context, req admission.Request) admission.Response {
+	obj := &unitv1alpha1.YurtAppDaemon{}
+
+	err := h.Decoder.Decode(req, obj)
+	if err != nil {
+		return admission.Errored(http.StatusBadRequest, err)
+	}
+
+	unitv1alpha1.SetDefaultsYurtAppDaemon(obj)
+	obj.Status = unitv1alpha1.YurtAppDaemonStatus{}
+
+	statefulSetTemp := obj.Spec.WorkloadTemplate.StatefulSetTemplate
+	deployTem := obj.Spec.WorkloadTemplate.DeploymentTemplate
+
```

Comment:
how about add verification that statefulset template and deployment template should not be exist at the same time.